src/_posts/compliance/2000-01-01-iso27001.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "NF EN ISO/CEI 27001:2017"
+nav: ISO 27001
+modified_at: 2022-09-12 00:00:00
+tags: compliance iso-27001
+---
+*(version française [ci-dessous](./2000-01-01-iso27001.md#fr-has-scalingo-iso-27001))*
```

Comment:
Same wrong URL

See https://github.com/Scalingo/documentation#dos `If you want to insert a link to another documentation article:`